### ISSUE TITLE
Bedrock: Cross Region Inference capability 

### ISSUE THREAD

**Issue by medha-hegde** [aws/aws-cdk#31570]
### Describe the feature

The [new](https://eu-central-1.console.aws.amazon.com/bedrock/home?region=eu-central-1#/cross-region-inference) cross-region inference profiles allow us to use the same Bedrock model in other regions by switching out the foundation model ARN with this inference profile ARN:
<img width="779" alt="image" src="https://github.com/user-attachments/assets/9c22adbb-1328-4c6b-a2f2-27ee475aaafd">

However, in the Amazon Bedrock Construct Library only allows for Foundation Models or Provisioned Models. 

Can we incorprate this new cross-region inference profiles feature ASAP? 

### Use Case

I am invoking the Bedrock API via Step Functions tasks with this code currently:

```
claude_model_id = bedrock.FoundationModelIdentifier.ANTHROPIC_CLAUDE_V2_1
invoke_model = tasks.BedrockInvokeModel(
        scope,
        id + " (Invoke Model)",
        model=bedrock.FoundationModel.from_foundation_model_id(
            scope,
            "Model",
            claude_model_id,
        ),...
```
ideally I'd like to simply switch out the `claude_model_id` for the inference profile ARN. 

### Proposed Solution

Allow for custom ARNs instead of enforcing the `bedrock.FoundationModelIdentifier` class for the model IDs. 

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.155.0

### Environment details (OS name and version, etc.)

MacOS Ventura, M2 Chip

**Comment by pahud** [comment]
Makes sense. We can consider to create a new static method for that. We welcome any PRs from the community.

**Comment by jdebuseamazon** [comment]
#31958 adds some cross regional inference constructs that might be useful
